examples/pxScene2d/src/Makefile

```diff
@@ -106,6 +109,7 @@ PX_SRCS_FULL=\
   pxImage9.cpp \
   pxImageA.cpp \
   pxArchive.cpp \
+  pxServiceManager.cpp \
```

Comment:
pxServiceManager.cpp should be conditionally added based on the SERVICE_MANAGER environment variable.  Please refer to the comment above.